src/Hangfire.Core/Dashboard/Pages/ScheduledJobsPage.generated.cs

```diff
@@ -282,23 +367,23 @@ Delete selected
 
 
             
-            #line 90 "..\..\Dashboard\Pages\ScheduledJobsPage.cshtml"
+            #line 130 "..\..\Dashboard\Pages\ScheduledJobsPage.cshtml"
                                  if (job.Value.ScheduledAt.HasValue)
                                 {
                                     
             
             #line default
             #line hidden
             
-            #line 92 "..\..\Dashboard\Pages\ScheduledJobsPage.cshtml"
-                               Write(Html.RelativeTime(job.Value.ScheduledAt.Value));
+            #line 132 "..\..\Dashboard\Pages\ScheduledJobsPage.cshtml"
+                               Write(Html.Raw(job.Value.ScheduledAt.Value.ToString("dd/MM/yyyy HH:mm")));
 
             
             #line default
             #line hidden
             
-            #line 92 "..\..\Dashboard\Pages\ScheduledJobsPage.cshtml"
-                                                                                   
+            #line 132 "..\..\Dashboard\Pages\ScheduledJobsPage.cshtml"
+                                                                                                       ;
```

Comment:
this file is generated from the .cshtml file through the razor tool. Everything looks a bit messed up in those files :x